django_countries/widgets.py

```diff
@@ -77,11 +77,7 @@ def render(self, name, value, attrs=None, renderer=None):
             )
         else:
             flag_id = ""
-        # Renderer argument only added in 1.11, keeping backwards compat.
-        kwargs = {"renderer": renderer} if renderer else {}
-        widget_render = super().render(
-            name, value, attrs, **kwargs
-        )
+        widget_render = super().render(name, value, attrs, renderer=renderer)
```

Comment:
Why not just send kwargs?

Reply:
It's more explicit, given that we know exactly what kwargs we are passing, and should anyone want to type hint it'll make it easier for them.